wp-content/themes/cpipr/lcdm-single.php

```diff
@@ -0,0 +1,199 @@
+                        $lang = has_tag('spanish') ? 'spanish' : 'english';
+                        $args = array(
+                            'post_type' => 'post',
+                            'tax_query' => array(
+                                array(
+                                    'taxonomy' => 'series',
+                                    'field'    => 'slug',
+                                    'terms'    => 'los-chavos-de-maria',
+                                ),
+                                array(
+                                    'taxonomy' => 'post_tag',
+                                    'field'    => 'slug',
+                                    'terms'    => $lang
+                                )
+                            ),
+                            'order' => 'DESC',
+                            'posts_per_page' => 3,
+                            'post_status' => 'publish',
+                            'post__not_in' => array(get_the_ID())
+                        );
+                        $chavos_query = new WP_Query($args);
+                        while ($chavos_query->have_posts()) {
+                            $chavos_query->the_post();
+                    ?>
+                    <div class="span4">
+                        <div class="card card-inphographic">
+                            <div class="card-image-top">
+                                <?php echo the_post_thumbnail('full') ?>
+                            </div>
+                            <div class="card-body">
+                                <h5 class="card-title"><a href="<?php the_permalink();?>"><?php the_title(); ?></a></h5>
+                                <p class="card-text card-post-date clearfix">
+                                    <?php echo get_the_date(); ?>
+                                    <a href="<?php the_permalink();?>" class="btn btn-white-blue pull-right">LEER MAS</a>        
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <?php } wp_reset_postdata();?>
+                    
+                </div>
+            </div>
+        </div>
+
+
+        <?php get_template_part('partials/los-chavos-de-maria/' . ($lang == 'spanish' ? 'es' : 'en') . '/footer'); ?>
+
+        <script type="text/javascript">
+            (function ($) {
+                var images = document.querySelectorAll('.wp-caption-img-wrap img');
+                new simpleParallax(images, {
+                    orientation: 'down',
+                    delay: 0.2,
+                    scale: 1.4
+                });
+
+                var heros = document.querySelectorAll('.owl-hero-image img');
+                new simpleParallax(heros, {
+                    orientation: 'down',
+                    delay: 0.2,
+                    scale: 1.4
+                });
+            })(jQuery);
+        </script>
+
+    </body>
```

Comment:
Is this template not going to use the standard footer? If it will, you can remove the ending `body` and `html` tags since the footer will take care of that for you.